packages/map/src/vaadin-map.js

```diff
@@ -63,7 +63,9 @@ class Map extends ResizeMixin(FocusMixin(ElementMixin(ThemableMixin(PolymerEleme
       <style>
         :host {
           display: block;
-          width: 100%;
+          height: 400px;
+          flex: 1 1 auto;
+          align-self: stretch;
```

Comment:
Aligned the flex properties with Grid. Not sure if we had issues with Grid in that regard, but they seem reasonable to have.